imbi/clients/opensearch.py

```diff
@@ -77,22 +80,27 @@ async def initialize(self) -> bool:
         self.client = opensearchpy.AsyncOpenSearch(
             **self.settings['connection'])
 
-        LOGGER.debug('Creating projects index in OpenSearch')
+        self.timer_handle = self.loop.call_soon(
+            lambda: asyncio.ensure_future(self._process()))
```

Comment:
You should be able to use `asyncio.create_task(self._process())` here. [ref](https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task)